src/main/java/ee/tuleva/onboarding/capital/CapitalService.java

```diff
@@ -5,64 +5,54 @@
 import ee.tuleva.onboarding.capital.event.member.MemberCapitalEvent;
 import ee.tuleva.onboarding.capital.event.member.MemberCapitalEventRepository;
 import ee.tuleva.onboarding.capital.event.member.MemberCapitalEventType;
-import ee.tuleva.onboarding.capital.event.organisation.OrganisationCapitalEventRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 import javax.validation.constraints.NotNull;
 import java.math.BigDecimal;
-import java.math.MathContext;
 import java.time.LocalDate;
 import java.util.List;
 
+import static ee.tuleva.onboarding.capital.event.member.MemberCapitalEventType.*;
+import static java.math.BigDecimal.ROUND_HALF_DOWN;
+
 @Service
 @RequiredArgsConstructor
 public class CapitalService {
 
     private final MemberCapitalEventRepository memberCapitalEventRepository;
-    private final OrganisationCapitalEventRepository organisationCapitalEventRepository;
     private final AggregatedCapitalEventRepository aggregatedCapitalEventRepository;
 
     CapitalStatement getCapitalStatement(Long memberId) {
         List<MemberCapitalEvent> events = memberCapitalEventRepository.findAllByMemberId(memberId);
 
-        BigDecimal capitalPaymentAmount =
-            getCapitalAmount(events, MemberCapitalEventType.CAPITAL_PAYMENT);
-
-        BigDecimal membershipBonusAmount =
-            getCapitalAmount(events, MemberCapitalEventType.MEMBERSHIP_BONUS);
-
-        BigDecimal unvestedWorkCompensationAmount =
-            getCapitalAmount(events, MemberCapitalEventType.UNVESTED_WORK_COMPENSATION);
-
-        BigDecimal workCompensationAmount =
-            getCapitalAmount(events, MemberCapitalEventType.WORK_COMPENSATION);
-
-
-       return new CapitalStatement(
-           membershipBonusAmount, capitalPaymentAmount,
-           unvestedWorkCompensationAmount, workCompensationAmount,
-           getProfit(events)
-       );
+        return new CapitalStatement(
+            getCapitalAmount(events, MEMBERSHIP_BONUS),
+            getCapitalAmount(events, CAPITAL_PAYMENT),
+            getCapitalAmount(events, UNVESTED_WORK_COMPENSATION),
+            getCapitalAmount(events, WORK_COMPENSATION),
+            getProfit(events)
+        );
     }
 
     @NotNull
     private BigDecimal getCapitalAmount(List<MemberCapitalEvent> events, MemberCapitalEventType capitalPayment) {
         return events.stream().filter(event -> event.getType() == capitalPayment)
-            .filter(event -> event.getAccountingDate().compareTo(LocalDate.now()) < 1)
```

Comment:
I would create a method here
```
Boolean isBeforeOrEqual(LocalDate first, LocalDate second){
return first.isBefore(second) || first.isEqual(second);
}
```

Reply:
Equal is probably an edge case (i.e. it has to be equal by the nanosecond - highly unlikely). I'll just use `isBefore()` for better readability.

Reply:
Scratch that, LocalDate has no time :)